src/encord_active/lib/common/utils.py

```diff
@@ -438,6 +434,22 @@ def download_file(
     return destination
 
 
+def download_image(url: str) -> Image.Image:
+    r = requests.get(url)
+
+    if r.status_code != 200:
+        raise ConnectionError(f"Something happened, couldn't download file from: {url}")
+
+    return Image.open(r.content)
+
+
+def convert_image_to_cv2(image: Image.Image) -> np.ndarray:
+    rgb_image = image.convert("RGB")
+    np_image = np.array(rgb_image)
+    ocv_image = np_image[:, :, ::-1].copy()
+    return ocv_image
```

Comment:
I would probably call this `convert_to_bgr` instead.
```suggestion
def convert_image_bgr(image: Image.Image) -> np.ndarray:
    rgb_image = image.convert("RGB")
    np_image = np.array(rgb_image)
    return np_image[:, :, ::-1].copy()
```

Reply:
☝🏻 Remember to update usages if you change the name ;-)